package.json

```diff
@@ -38,8 +38,10 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@zowe/cli": "^7.16.6",
     "@zowe/imperative": "5.15.1",
-    "@zowe/perf-timing": "1.0.7"
+    "@zowe/perf-timing": "1.0.7",
+    "cd": "^0.3.3"
```

Comment:
Is the "cd" package used in this PR? It appears to be over 10 years and old and no longer maintained, so wondering if it should be removed from the "dependencies" list here in package.json